reporters/SponsorReporter.js

```diff
@@ -65,43 +85,13 @@ class SponsorReporter {
         "Collateral(DAI)": this.formatDecimalString(position.amountCollateral),
         "Tokens borrowed(ETHBTC)": this.formatDecimalString(position.numTokens),
         "Position CR %": this.formatDecimalString(
-          this._calculatePositionCRPercent(position.amountCollateral, position.numTokens, currentPrice).muln(100)
```

Comment:
This is a good refactor. Moving technical details away from client is a good practice IMO.